docs/advanced-features/workflows.md

```diff
@@ -185,9 +185,19 @@ quilt3.Package().set_meta({'answer': 42}).push(
 # Package test/package@6331508 pushed to s3://quilt-sergey-dev-metadata
 ```
 
-If you wish for your users to be able to skip workflows altogether, you can make
-workflow validation optional with `is_workflow_required: false` in your `config.yml`,
-and specify `workflow=None` in the API:
+## Bypassing workflow validation and setting a default workflow
```

Comment:
My intended goal in requesting the update to this document was to help users understand how to require workflows. Though the inverse is also helpful, I'd like to help users understand how to enforce workflows using `is_workflow_required: True`, and how to best utilize this. 

If `is_workflow_required` is not explicitly called out, what is the default state?

Reply:
By default workflows are required. We should state that explicitly.

Reply:
Fixed!